frontend/src/views/organisation/workflows/MoOrganisationUnitCreate.vue

```diff
@@ -115,14 +115,16 @@ export default {
         let vm = this
         this.isLoading = true
 
-        this.addresses.push(this.postAddress, this.phone)
-        this.addresses.forEach(a => {
-          if (!a.validity) {
-            a.validity = this.entry.validity
-          }
-          a.org = this.$store.getters['organisation/GET_ORGANISATION']
-        })
-        this.entry.details = this.addresses
+        if (!vm.backendValidationError) {
+          this.addresses.push(this.postAddress, this.phone)
```

Comment:
From my understanding, `backendValidationError` gets set whenever a request to the backend fails. Once this occurs, you will no longer store that new addresses; correct?

It just so happens that this may work due to the exact instances of `this.postAddress` and `this.phone` appearing in the array, and Vue merely updates those. That being said, I think it would be cleaner to simply reassign `this.addresses`, or explicitly clearing it prior to the assignment.